tests/functional/api/test_releases.py

```diff
@@ -23,6 +23,17 @@ def test_create_project_release(project, project_file):
     assert release.description == release_description
 
 
+def test_update_save_project_release(project, release):
+
+    release = project.releases.get(release_tag_name)
+    release.description = release_description + "updated"
+    release.save()
+
+    assert project.releases.get(release_tag_name)
+    assert release.tag_name == release_tag_name
+    assert release.description == release_description + "updated"
```

Comment:
Since we have a function-scoped fixture for this (https://github.com/python-gitlab/python-gitlab/blob/master/tests/functional/conftest.py#L318-L332), I think it's best to use it instead of reusing one from previous tests, as we then introduce a dependency between them.

You are already passing it in the test but override it. See how we also use the release fixture in subsequent tests. Maybe something like this, removing the initial `get()`:
```suggestion
    updated_description = f"{release.description} updated"
    release.description = updated_description
    release.save()

    release = project.releases.get(release.tag_name)
    assert release.description == updated_description
```

Reply:
I'll try to look at it today.
Otherwise it will be in 2 weeks! Time for 🌊 after this long year!

Reply:
Sure! If you can't make it this weekend I can merge this as is and tweak tests in a follow-up, it's not blocking.

Enjoy your holidays! :)

Reply:
Change done :)
Thanks for the review :)